### DISCUSSION TITLE
Color Change

### DISCUSSION THREAD

**Discussion by SSUzunov19** [steveruizok/perfect-freehand#64] [Q&A]
How to change color of the pencil?

**Comment by codezerro** [comment]
no one answered your question. have you find any solution?

**Comment by nate-lumalabs** [comment]
It depends how you are rendering your lines. If you are rendering them as SVG shapes, you can use the svg fill property.

```
const myStroke = getStroke(points, options);
const pathData = getSvgPathFromStroke(myStroke);
...
<svg>
<path fill="red" d={myPathData} />
</svg>
```
